packages/cli/src/scaling/scaling.service.ts

```diff
@@ -55,6 +55,8 @@ export class ScalingService {
 	// #region Lifecycle
 
 	async setupQueue() {
+		if (this.queue) return;
```

Comment:
I'm not 100% but I think checking the queue here could lead to race condition, because the code could enter this method twice while we wait in line 60 or 61 for the dynamic imports, in which case the queue value would be overwritten by the later call. If we move this check after the await statements in line 60 and 61 the assignment of the queue variable will only happen once, because there no real threads.